hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java

```diff
@@ -6600,13 +6601,11 @@ public void test_CPSubsystemAddGroupAvailabilityListenerCodec_decodeResponse() {
 
     private static class CPSubsystemAddGroupAvailabilityListenerCodecHandler extends CPSubsystemAddGroupAvailabilityListenerCodec.AbstractEventHandler {
         @Override
-        public void handleGroupAvailabilityEventEvent(com.hazelcast.cp.internal.RaftGroupId groupId, java.util.Collection<com.hazelcast.cp.CPMember> members, java.util.Collection<com.hazelcast.cp.CPMember> unavailableMembers,
-                                                      boolean isIsShutdownExists, boolean isShutdown) {
+        public void handleGroupAvailabilityEventEvent(com.hazelcast.cp.internal.RaftGroupId groupId, java.util.Collection<com.hazelcast.cp.CPMember> members, java.util.Collection<com.hazelcast.cp.CPMember> unavailableMembers, boolean isIsShutdownExists, boolean isShutdown) {
             assertTrue(isEqual(aRaftGroupId, groupId));
             assertTrue(isEqual(aListOfCpMembers, members));
             assertTrue(isEqual(aListOfCpMembers, unavailableMembers));
-            assertFalse(isIsShutdownExists); // < 2.7
```

Comment:
most changes are expected header changes, but on this file and on all other tests that have changed... is that an expected change?

Reply:
> most changes are expected header changes, but on this file and on all other tests that have changed... is that an expected change?

yes and no - they were last changed in c15043352a2ec157ccf36458e767c935ecadcafb, but it looks like they were manually modified on top of being automatically generated (e.g. difference in formatting, comments etc).
Then when I've regenerated the code, those manual changes have been undone showing a difference.
It's only a change to the tests and they still pass, so I'm happy.

It'd probably be "cleaner" to regenerate the code from the previous commit to fix that, and then make this change, but I'm not sure that's worth it.

Reply:
The files are not expected to be modified anyway. Let's ignore the changes.